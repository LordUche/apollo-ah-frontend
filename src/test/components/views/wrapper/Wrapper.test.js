import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Wrapper from '../../../../components/views/common/Wrapper';

configure({ adapter: new Adapter() });

describe('<Wrapper>', () => {
  it('should render ', () => {
    const container = (
      <Wrapper>
        <p>Hello</p>
      </Wrapper>
    );
    const wrapper = shallow(container);
    expect(wrapper).toMatchSnapshot();
  });
});
