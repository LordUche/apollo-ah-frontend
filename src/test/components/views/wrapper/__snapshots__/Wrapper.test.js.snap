// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Wrapper> should render  1`] = `
<main
  className="main-body"
>
  <section
    className="wrapper min-vh-100"
  >
    <div
      className="container"
    >
      <p>
        Hello
      </p>
    </div>
  </section>
</main>
`;
